include/net/download_client.h

```diff
@@ -117,118 +154,72 @@ struct download_client_cfg {
 	 * Zero is the default PDN.
 	 */
 	uint8_t pdn_id;
-	/**
-	 * Address family to be used for the download, AF_INET6 or AF_INET.
-	 * Set to AF_UNSPEC (0) to fallback to AF_INET if AF_INET6 does not work.
-	 */
-	int family;
 	/** Maximum fragment size to download. 0 indicates that values
 	 * configured using Kconfig shall be used.
```

Comment:
Write down with @kconfig{CONFIG_DOWNLOAD_CLIENT_THIS_OPTION} the options.

Reply:
Could you please elaborate what you mean here?

Reply:
> Ah, you mean to specify the Kconfig that is used.

Yep, say the exact Kconfig options that are used when this value is 0.